src/Standards/Squiz/Tests/WhiteSpace/LanguageConstructSpacingUnitTest.inc

```diff
@@ -30,3 +30,10 @@ return;
 return $blah;
 return   $blah;
 return($blah);
+
+return	$tab;
+return
+$newLine;
+
```

Comment:
if this is gonna be merged I believe it should cover one more test case. What happens if there is a single space after return and then a new line.